### DISCUSSION TITLE
My PCB Build of the OneInchEye

### DISCUSSION THREAD

**Discussion by yassinebkr** [will127534/OneInchEye#26] [Show and tell]
Hello everyone,

I'm excited to share with you my assembled PCB version of the [OneInchEye](https://github.com/will127534/OneInchEye/) project. After following the schematics and instructions from the repository, I was able to successfully build and test the device. And guess what ? Fully working !
The luckiest part was being able to find some imx283.
thanks to will whang and all this beautiful community.
![PXL_20241109_231627721](https://github.com/user-attachments/assets/80eebb46-e994-4498-b0f9-c653b8b841e6)
![PXL_20241109_231547721](https://github.com/user-attachments/assets/47054a9d-efb0-4238-b0a9-4e17aff2e704)
![Screenshot_20241110-001005](https://github.com/user-attachments/assets/894d5e6e-e356-4696-8719-aaafcf78ce83)
![Screenshot_20241110-000945](https://github.com/user-attachments/assets/db30a4a2-751e-4d28-a86a-1ca572df0aaa)

**Comment by will127534** [comment]
Congrats!

**Comment by 230021-netizen** [comment]
what'd you use to hold the pcb and sensor together?
